devicetypes/smartthings/zigbee-switch.src/i18n/messages.properties

```diff
@@ -0,0 +1,17 @@
+#  Copyright 2019 SmartThings
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy
+#  of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+
+# Chinese
+'''Orvibo Smart Switch'''.zh-cn=智能墙面开关(一路版)
+'''Orivbo Smart Outlet'''.zh-cn=智能插座
```

Comment:
Should be `Orvibo`

Reply:
https://smartthings.atlassian.net/browse/ICP-10321

Reply:
@FenMei007S please double check that ticket. That one doesn't appear to have anything to do with this device.

Reply:
Dear tom, sorry for make confused, it is ICP-10286